test/support/annotations/vcr_request.rb

```diff
@@ -0,0 +1,34 @@
+module Annotations
+  class VcrRequest
+    attr_accessor :method, :path, :request_body, :response_body, :response_code
+
+    def initialize(hash)
```

Comment:
Consider using `fetch` on the hash to ensure each of the values below is defined.